src/cljs/rems/administration/workflow.cljs

```diff
@@ -65,6 +66,15 @@
                     [inline-info-field (str (text :t.create-license/external-link)
                                             " "
                                             (str/upper-case (:langcode localization))) [:a {:target :_blank :href (:textcontent localization)} (:textcontent localization) " " [external-link]]]))
+                (when (= "attachment" (:licensetype license))
+                  (for [localization (:localizations license)]
+                    (when (:attachment-id localization)
+                      [inline-info-field
+                       (str (text :t.create-license/license-attachment)
+                            " "
+                            (str/upper-case (:langcode localization)))
+                       [attachment-link (:attachment-id localization) (:title localization)]
+                       {:no-box? true}])))
```

Comment:
Could all the new code above be unified with resource licenses if you gave the text as a parameter to a function that both would use?

Reply:
Yeah that's what the TODO is there for. It's unclear at the moment if resource, workflow or form licenses all are needed, and if they are all equal. Also the structures that our API returns are currently not identical so they should be unified. This however is quite a lot of work that I'd rather do later when the first questions are resolved.

Reply:
I was thinking just something like this:

```
(defn create-license-fields [info-text entity]
  (for [localization (:localizations entity)]
                    (when (:attachment-id localization)
                      [inline-info-field
                       (str info-text " " (str/upper-case (:langcode localization)))
                       [attachment-link (:attachment-id localization) (:title localization)]
                       {:no-box? true}])))

....
(when (= "attachment" (:licensetype license))
    (create-license-fields (text :t.create-license/license-attachment) license)
...

```